internal/cluster/transition.go

```diff
@@ -341,3 +341,29 @@ func setPendingUserReset(ctx context.Context, c *common.Cluster, db *gorm.DB, ho
 	txSuccess = true
 	return nil
 }
+
+func (th *transitionHandler) PostRefreshClusterValidationsInfoUpdate(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) error {
```

Comment:
why we need another method PostRefreshClusterValidationsInfoUpdate - PostRefreshCluster method also update the validation_info in the DB?